### PR TITLE
LimitedAds: add documentation

### PR DESCRIPTION
**Pull request by sl-pierrevautherin**
https://github.com/jaanauati/react-dfp/pull/209

### CODE REVIEW

File: api.json [hunk 1]
```diff
@@ -1,55 +1,72 @@
...
+    "shouldRefresh": {}
+  },
+  "DFPSlotsProvider": {
+    "dfpNetworkId": {},
+    "personalizedAds": {},
+    "cookieOption": {},
+    "singleRequest": {},
+    "disableInitialLoad": {},
+    "lazyLoad": {},
+    "limitedAds": {},
```
Comment (sl-pierrevautherin) [inline api.json:36]: I only added the line ⬆️ 
The rest is just formatting...
I couldn't unstage this changes

File: components/ads/limitedAds/example.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { DFPSlotsProvider, AdSlot } from 'react-dfp';
+
+export default class Example extends Component {
+  render() {
+    return (
+      <DFPSlotsProvider
+        limitedAds
```
Comment (sl-pierrevautherin) [inline components/ads/limitedAds/example.js:8]: The new param we added in PR #209

File: components/ads/limitedAds/intro.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React, { Fragment } from 'react';
+
+export default function () {
+  return (
+    <Fragment>
+      <p>
+        <strong>New !!</strong> please upgrade to version >= 0.20.0 to use this feature
```
Comment (sl-pierrevautherin) [inline components/ads/limitedAds/intro.js:7]: I'm happy to change the wording here 😅

**Comment by jaanauati** [comment]
LGTM, thank you!

**Pull request merged**